Mrs_Cake/Controllers/UsersController.cs

```diff
@@ -31,13 +33,13 @@ public ActionResult<List<User>> Get()
         [HttpGet("{id:length(24)}", Name = "GetUser")]
         public ActionResult<User> GetById(string id)
         {
-            var userFromDB = _userService.GetById(id);
+            var usersFromDB = _userService.GetById(id);
```

Comment:
Here you get just one user, because you do "get user by id", so usersFromDB is not a good name for variable

Reply:
I will look into it in next commit.